unit-tests/run-unit-tests.py

```diff
@@ -290,6 +299,8 @@ def __init__( self, source, line_prefix ):
                     self._priority = int( params[0] )
                 else:
                     log.e( source + '+' + str(context['index']) + ': priority directive with invalid parameters:', params )
+            elif directive == 'tag':
+                self._tags.add(params)

```

Comment:
```
>>> s = set()
>>> p = ['1', '2']
>>> s.add( p )
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unhashable type: 'list'
```
Will this work? Am I doing something wrong?

Reply:
I thought so but apparently only works with immutable. I'll change it to use update instead